README.md

```diff
@@ -1,80 +1,86 @@
 # Datadog Cloud Foundry Buildpack
 
-This is a [decorator buildpack](https://github.com/cf-platform-eng/meta-buildpack/blob/master/README.md#decorators) for Cloud Foundry. It will install a Datadog DogStatsD binary in the container your app is running on. This only contains the DogStatsd components of Datadog Agent, which has limited overhead.
+This is a [supply buildpack](https://docs.cloudfoundry.org/buildpacks/understand-buildpacks.html#supply-script) for Cloud Foundry. It installs the following three binaries in the container your application is running on:
+* Datadog Dogstatsd for submitting custom metrics from your application
+* Datadog Trace Agent for submitting APM traces from your application
+* Datadog IoT Agent for submitting your application logs
 
-## Use
-
-### Install the [Meta Buildpack](https://github.com/cf-platform-eng/meta-buildpack#how-to-install-the-meta-buildpack)
-First, you will have to [install the Meta Buildpack](https://github.com/cf-platform-eng/meta-buildpack#how-to-install-the-meta-buildpack). This enables apps to use decorator buildpacks. Follow the instructions to get the buildpack and upload it if you don't already have it.
+## Installation
 
 ### Upload the buildpack to CF
-- Download the latest Datadog [build pack release](https://cloudfoundry.datadoghq.com/datadog-cloudfoundry-buildpack/datadog-cloudfoundry-buildpack-latest.zip) or [build it](/DEVELOPMENT.md#building). After you have the zipfile, you will have to upload it to Cloud Foundry environment.
 
-- Create the buildpack in CF if it does not exist
+Download the latest Datadog [buildpack release](https://cloudfoundry.datadoghq.com/datadog-cloudfoundry-buildpack/datadog-cloudfoundry-buildpack-latest.zip) or [build it](/DEVELOPMENT.md#building) and upload it to your Cloud Foundry environment.
+
+- Upload buildpack for the first time
     ```bash
     cf create-buildpack datadog-cloudfoundry-buildpack datadog-cloudfoundry-buildpack.zip 99 --enable
     ```
-    or update it if it already exists
+- Update existing buildpack
     ```bash
     cf update-buildpack datadog-cloudfoundry-buildpack -p datadog-cloudfoundry-buildpack.zip
     ```
+Once it is available in your Cloud Foundry environment, configure your application to use the Datadog buildpack by specifying it in your application manifest.
+
+**Note:** since this is a supply buildpack, it has to be specified before any final buildpack in the list. See [here](https://docs.cloudfoundry.org/buildpacks/use-multiple-buildpacks.html) for details about pushing an application with multiple buildpacks.
```

Comment:
```suggestion
**Note**: Since this is a supply buildpack, it has to be specified before any final buildpack in the list. See [Cloud Foundry documentation](https://docs.cloudfoundry.org/buildpacks/use-multiple-buildpacks.html) for details about pushing an application with multiple buildpacks.
```